src/coreclr/jit/fgopt.cpp

```diff
@@ -1279,11 +1293,11 @@ BasicBlock* Compiler::fgIntersectDom(BasicBlock* a, BasicBlock* b)
     BasicBlock* finger2 = b;
     while (finger1 != finger2)
     {
-        while (finger1->bbPostOrderNum > finger2->bbPostOrderNum)
+        while (finger1->bbPostorderNum < finger2->bbPostorderNum)
```

Comment:
Note this method is now very similar to the SSA builder's `IntersectDom` -- the main difference being how nodes with no predecessors are handled.